### PR TITLE
Update LICENSE.txt

### PR DESCRIPTION
**Pull request by davidcherney**
gave yall the truth

### CODE REVIEW

**Review by hamzakhokhar — APPROVED** [review]
done

**Review by evgeniya1 — COMMENTED** [review]
Not approved ;(

File: LICENSE.txt [hunk 1]
```diff
@@ -1,8 +1 @@
-asasdfasdfaasdfasdfasdfa
-
-
-
-NO wait... its 
-
-
-asdfasdfasfsadfasdfasdfasdf boom
\ No newline at end of file
+when you eat a doughnut you die.
```
Comment (evgeniya1) [inline LICENSE.txt:1]: Please capitalize the sentence.

**Review by evgeniya1 — CHANGES_REQUESTED** [review]
Please capitalize the sentence.

File: LICENSE.txt (hunk 1 above)
Comment (davidcherney) [reply LICENSE.txt:1]: ok

Comment (davidcherney) [reply LICENSE.txt]: ok, done, ok....

**Review by evgeniya1 — APPROVED** [review]
Great job!

**Pull request merged**